code/modules/projectiles/guns/pistols.dm

```diff
@@ -75,6 +75,47 @@
 	recoil = -2
 	recoil_unwielded = -2
 
+//-------------------------------------------------------
+//TX-7 Plasma Pistol
+/obj/item/weapon/gun/pistol/plasma_pistol
+	name = "\improper TX-7 Plasma Pistol"
+	desc = "An expiramental weapon. Designed to set the terrain it is fired at on fire so as to remove flammable obstacles and clear out dug in enemies from some range."
```

Comment:
Typo here, that should be 'experimental'  instead.

Reply:
thank you for noticing and drawing light to my terrible spelling.